bound/kt/src/main/kotlin/tbdex/sdk/web5/KeyManager.kt

```diff
@@ -0,0 +1,11 @@
+package tbdex.sdk.web5
+
+import tbdex.sdk.rust.JwkData as RustCoreJwk
+import tbdex.sdk.rust.KeyManager as RustCoreKeyManager
+
+typealias Jwk = RustCoreJwk
+
+interface KeyManager {
+    fun getSigner(publicJwk: Jwk): Signer
+    fun getRustCoreKeyManager(): RustCoreKeyManager
```

Comment:
this goes against our APID right?

Reply:
it does, but as far as I can tell it's necessary to make polymorphism work... we need to iterate on it because we want developers to be able to bring their own key manager's in which case this method would error-out. We'll come back to it later to add support, once we add `WithForeign` in the UDL layer.